packages/botonic-react/src/webchat/hooks.js

```diff
@@ -51,7 +51,7 @@ export const webchatInitialState = {
   },
   themeUpdates: {},
   error: {},
-  devSettings: {},
+  devSettings: { keepSessionOnReload: false },
```

Comment:
maybe you can add in the PR description a line mentioning this change?

Reply:
added 👍